src/MinimumNodeService.cpp

```diff
@@ -139,10 +139,23 @@ void MinimumNodeService::process(const Action *action)
         break;
       
       case ACT_RENEGOTIATE:
-        switch (controller->getModuleConfig()->currentMode)
+        switch (instantMode)
         {
         case MODE_UNINITIALISED:
           break;
+          
+        case MODE_SETUP:
```

Comment:
Please add a comment that this action cancels the setup and reverts to previous mode.

Reply:
Done and other change we discussed.